### PR TITLE
adds mining scanners to the gulag

### PR DESCRIPTION
**Pull request by ghost**
[Changelogs]: # (Your PR should contain a detailed changelog of notable changes, titled and categorized appropriately. This includes, new features, sprites, sounds, balance changes, admin tools, map edits, removals, big refactors, config changes, hosting changes and important fixes. An example changelog has been provided below for you to edit. If you need additional help, read https://github.com/tgstation/tgstation/wiki/Changelogs)

:cl: Awesine
add: added some things which happen to be mining scanner things to that gulag thing
/:cl:

[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.)

muh  >in order to get 100 points for the gulag you have to mine through a bazillion miles of rock and god heil if you hit minerals because you have no idea where they are

### CODE REVIEW

File: _maps/map_files/Mining/Lavaland.dmm [hunk 1]
```diff
@@ -111,6 +111,7 @@
 /obj/item/pickaxe,
 /obj/item/device/flashlight,
 /obj/item/clothing/glasses/meson,
+/obj/item/device/t_scanner/adv_mining_scanner,
```
Comment (DaxDupont) [inline _maps/map_files/Mining/Lavaland.dmm]: >advanced mining scanner

Comment (awesine) [reply _maps/map_files/Mining/Lavaland.dmm]: yeah what

**Comment by ShiggyDiggyDo** [comment]
Gulag is a punishment, not a super fun mining experience without monsters

**Comment by awesine** [comment]
@ShiggyDiggyDo 
gulag was added when iron ore were visible, thus making a balance

now they aren't

now it's actually lucky if you can fucking find ore

have you ever actually been to the gulag


probably not because coderbus doesn't play but i can still ask

**Comment by GunHog** [comment]
Give the Gulag a **manual** mining scanner. Criminal scum do not deserve convenience.

**Comment by awesine** [comment]
@GunHog 
STOP RIGHT THERE, CODERBUS SCUM!

**Comment by awesine** [comment]
okay for images
image https://i.imgur.com/yi9pgiJ.png

but the racks have adv mining scanners

**Comment by Tlaltecuhtli** [comment]
uh so if i m a miner i can easly go break into gulag and steal their scanners and use my voucher on cool mining drones

**Comment by awesine** [comment]
@Tlaltecuhtli 
yeah sure ok

**Comment by Bawhoppen** [comment]
Resourcecreep 101

**Review by ShizCalev — DISMISSED** [review]

**Comment by lzimann** [comment]
Just make it the manual scanner and it will be fine.

**Comment by awesine** [comment]
can't change them from manual scanners right now because i gotta go to a christmas thing but i will when i get back

**Comment by FantasticFwoosh** [comment]
Mass digging labour **is part of the punishment**

If you give them a scanner they will beeline towards the rare valuable minerals & fufill all but the most absurd point sentences almost immediately by prioritizing ores like titanium & uranium.

**Comment by DaxDupont** [comment]
Sounds like we need to double the max point cap

**Comment by deathride58** [comment]
You can hit the max point cap for gulag so easily it's not even funny, and breaking out via running across lava is even easier if you're a human or if you're at full health.

Last I was gulagged, it took me five minutes flat to hit the point cap of 1000 points. With this PR in place, that'd make getting out via labor much faster, even if the scanner were changed to the manual variant.

The point of gulag is to be an alternative to perma, but with how piss-easy it is to get out through both ICly legal and ICly illegal ways, the only actual use of throwing someone in gulag is to see how robust they are.

**Comment by lzimann** [comment]
Gulag is not supposed to be an alternative to perma, it even has a limited max amount of points you can set, which means the person can always/almost always return to the station.

**Comment by DaxDupont** [comment]
It should take more than 5 minutes though.

**Comment by AdamElTablawy** [comment]
DIG THE FUCKING SAND YOU IDIOT

**Comment by awesine** [comment]
https://i.imgur.com/0ziCjJP.png

changing to manual scanners now should be up soon

**Comment by epochayur** [comment]
just do manual and crank the max points, if I'm gonna be stuck in the gulag for a majority of the round, I'd at least like to have something to do

**Comment by awesine** [comment]
replaced adv scanners with manual scanners

**Review by ShizCalev — APPROVED** [review]

**Comment by cebutris** [comment]
Will this allow them to get gibonite?

**Comment by awesine** [comment]
yeah mining scanners do that

**Comment by deathride58** [comment]
>somewhat lengthy escape
It takes less than a minute to escape the gulag, from the moment you're sent through the teleporter to the moment you touch the mining outpost's shuttle console. It's not that lengthy at all.

It's already easy and fast enough to get out of gulag via labor in the first place, there's no need to make it even easier. The only time getting points in the gulag is actually challenging is if sec has sent too many prisoners to the gulag, but due to the gulag rarely ever being used, this isn't an issue.

Giving prisoners effortless access to explosives is also bad. The content of this PR is what's known as powercreep.

**Comment by awesine** [comment]
@deathride58 
the problem is 90% of gulag players either just die or suicide as soon as they get there

**Comment by Repukan** [comment]
Powercreep and allows prisoners to receive explosives with no effort.

**Comment by Repukan** [comment]
@Ergovisavi I think this entails lavaland so what'd you think.

**Comment by FantasticFwoosh** [comment]
> @epochayur  : Just do manual and crank the max points, if I'm gonna be stuck in the gulag for a majority of the round, I'd at least like to have something to do. 

### You should have thought about that before you got caught & gulagged for being shit.

- Wardens are still going to remove the mining scanners from gulag anyway i can most certainly agree even with a point increase.

**Comment by awesine** [comment]
@FantasticFwoosh 
>security
>actually going TO the gulag
>knowing relevant code changes

you almost got me

**Comment by 81Denton** [comment]
>prisoners can now obtain gibtonite
That way it's now safer to just execute them

**Comment by jughu** [comment]
Put a 5 second delay on manuel scanners so either they disable the gibo immediatly or find out BOOM

**Comment by jughu** [comment]
100 points is like 30 seconds sand digging

**Comment by jughu** [comment]
If anything gulag is a faster brig sentence and it distrubutes to the station material wise

**Comment by Tlaltecuhtli** [comment]
where do the gulag materials go?

**Comment by OmegaShoots** [comment]
Make it a unique variant mining scanner called prisoner scanner that can't stabilize gibtonite and with a limited range of like 2-3(maybe more?) blocks and a hefty cooldown (30 seconds or so).

**Comment by jughu** [comment]
@tlalecuhtli the stacking machine but its extremely snowflake and doesnt drop its materials unless you get an full stack

**Comment by FantasticFwoosh** [comment]
@awesine they still have a gulag shuttle to just ride out before they actually send anybody there via the teleportation gulagger

If this PR passes it'll honestly just be a  case that X player doesn't like gulag in general but is too cowardly to start a pr to remove it so will undermine how it works first so it becomes irrelevant over time.

**Comment by Hatfish** [comment]
If you can't do the time, don't do the crime.

**Comment by FantasticFwoosh** [comment]
fuuuuuuuuugg

**Pull request merged**